prow/e2e-suite-rbac-auth.sh

```diff
@@ -28,7 +28,9 @@ set -u
 # Print commands
 set -x
 
-RBAC_FILE='install/kubernetes/istio-rbac-beta.yaml'
-
 echo 'Running e2e with rbac, with auth Tests'
-./prow/e2e-suite.sh --rbac_path="${RBAC_FILE}" --auth_enable "${@}"
+if [ "$#" != 0 ]; then
```

Comment:
Why is this required ?

Reply:
on Mac OSX, if the script is run with zero arguments, it would otherwise produce an error "Unbound variable".

Reply:
that sounds like a different problem, it shouldn't be necessary - what error exactly / do you have a small script showing the issue ?

Reply:
Try to run the original script without argument on MAC.

Reply:
works for me
we should avoid adding code that doesn't have an understood reason for being there, it just accumulates tech debt